packages/icons/web/v2/controls_16.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><clipPath id="a"><path d="m0 0h16v16h-16z"/></clipPath><g clip-path="url(#a)" fill="currentColor"><path d="m10 4.5c0 .55228.4477 1 1 1s1-.44772 1-1v-.5h3c.5523 0 1-.44772 1-1s-.4477-1-1-1h-3v-.5c0-.552285-.4477-1-1-1s-1 .447715-1 1z"/><path d="m1 2h7v2h-7c-.552285 0-1-.44772-1-1s.447715-1 1-1z"/><path d="m4 9.5c0 .5523.44772 1 1 1s1-.4477 1-1v-3c0-.55228-.44772-1-1-1s-1 .44772-1 1v.5h-3c-.552285 0-1 .44772-1 1 0 .55229.447715 1 1 1h3z"/><path d="m15 9h-7v-2h7c.5523 0 1 .44772 1 1 0 .55229-.4477 1-1 1z"/><path d="m11 15.5c-.5523 0-1-.4477-1-1v-3c0-.5523.4477-1 1-1s1 .4477 1 1v.5h3c.5523 0 1 .4477 1 1s-.4477 1-1 1h-3v.5c0 .5523-.4477 1-1 1z"/><path d="m8 12h-7c-.552285 0-1 .4477-1 1s.447715 1 1 1h7z"/></g></svg>
```

Comment:
We try to avoid ids in icons since they can cause issues when an icon is used multiple times on a page. I inspected this icon in Figma and found that the clip-path can be removed. Here's a version of the icon without it: 

```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path
    d="M10 4.5a1 1 0 0 0 2 0V4h3a1 1 0 1 0 0-2h-3v-.5a1 1 0 0 0-2 0v3ZM1 2h7v2H1a1 1 0 0 1 0-2Zm3 7.5a1 1 0 1 0 2 0v-3a1 1 0 0 0-2 0V7H1a1 1 0 0 0 0 2h3v.5ZM15 9H8V7h7a1 1 0 1 1 0 2Zm-4 6.5a1 1 0 0 1-1-1v-3a1 1 0 0 1 2 0v.5h3a1 1 0 0 1 0 2h-3v.5a1 1 0 0 1-1 1ZM8 12H1a1 1 0 0 0 0 2h7v-2Z"
    fill="currentColor" />
</svg>
```

Reply:
oh thank you! How you removed it? :O